### DISCUSSION TITLE
Brew directories not writable by user: can't install Scala

### DISCUSSION THREAD

**Discussion by ghost** [Homebrew/discussions#3805] [Everyday usage]
I'm trying to install scala but brew is throwing this error. It wants me to change ownership of these files and assign write access, but when I try it says my user name is illegal. But I am the only user on this macbook and the one who originally installed Homebrew. 

> Error: The following directories are not writable by your user:
> /usr/local/lib/pkgconfig
> /usr/local/share/info
> /usr/local/share/man/man3
> /usr/local/share/man/man5

Mac OS Monterey
Version 12.2.1
Homebrew 3 .6 .5

**Comment by SMillerDev** [comment]
Try running `sudo chown -R $(whoami) /usr/local/lib/pkgconfig` and see if it resolves that one.

**Reply by ghost** [reply]
> sudo chown -R $(Mando) /usr/local/lib/pkgconfig

Error: unexpected symbol in "sudo chown"

**Reply by SMillerDev** [reply]
That's not the command I gave though

**Reply by ghost** [reply]
doh! Ok, typed in exactly as you had. It prompted for password. Seems ok, but then I try to install 

brew install apache-spark
Error: The following directories are not writable by your user:
/usr/local/share/info
/usr/local/share/man/man3
/usr/local/share/man/man5

You should change the ownership of these directories to your user.
  sudo chown -R $(whoami) /usr/local/share/info /usr/local/share/man/man3 /usr/local/share/man/man5

And make sure that your user has write permission.
  chmod u+w /usr/local/share/info /usr/local/share/man/man3 /usr/local/share/man/man5

**Reply by SMillerDev** [reply]
Okay, that seems to have fixed the first permission. Now you can run that command with the other paths.

**Reply by ghost** [reply]
Brilliant, it worked!
